modules/apps/foundation/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/session.js

```diff
@@ -505,6 +505,7 @@ AUI.add(
 						instance._intervalId = host.registerInterval(
 							function(elapsed, interval, hasWarned, hasExpired, warningMoment, expirationMoment) {
 								if (!hasWarned) {
+									host.set('sessionState', 'active', SRC_EVENT_OBJ);
 									instance._uiSetActivated();
```

Comment:
does it still work with this line be removed ?

Reply:
I was recheck, it still works. It means I just need to change "sessionState " to "active" and the notification popup auto-hide..
Thank you.